Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj

```diff
@@ -85,6 +89,9 @@
       <HintPath>..\packages\System.Net.Http.Formatting.Extension.5.2.3.0\lib\System.Net.Http.Primitives.dll</HintPath>
       <Private>True</Private>
     </Reference>
+	    <Reference Include="System.Web.Http, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
added <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net462" /> to packges